frontend/src/components/pages/ItemsPage.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
+import './../../App.css';
```

Comment:
remove unnecessary import, we don't have to import App.css in any component, App.css is global